solutions/banking/outputs.tf

```diff
@@ -0,0 +1,44 @@
+output "project_id" {
+  description = "ID of the created project."
+  value       = module.configure_project.project_id
+}
+
+output "project_url" {
+  description = "Project ID URL."
+  value       = "https://dataplatform.cloud.ibm.com/projects/${module.configure_project.project_id}"
+}
+
+output "watsonx_assistant_url" {
+  description = "WastonX Assistant URL."
+  value       = local.watsonx_assistant_url
+}
+
+output "watsonx_discovery_url" {
+  description = "WastonX Discovery URL."
+  value       = local.watsonx_discovery_url
+}
+
+output "cos_instance_guid" {
+  description = "COS instance GUID."
+  value       = module.cos.cos_instance_guid
+}
+
+output "cos_instance_crn" {
+  description = "COS instance CRN."
+  value       = module.cos.cos_instance_crn
+}
+
+output "cos_instance_id" {
+  description = "COS instance ID."
+  value       = module.cos.cos_instance_id
```

Comment:
Not clear why we output both id and guid. Which one is typically used?

Reply:
removed both `id` and `guid` as CRN contains all required information. 

```
cos_instance_crn = "crn:v1:bluemix:public:cloud-object-storage:global:a/abac0df06b644a9cabc6e44f55b3880e:db530b95-6f50-4144-b0d0-6c341e23855f::"
cos_instance_guid = "db530b95-6f50-4144-b0d0-6c341e23855f"
cos_instance_id = "crn:v1:bluemix:public:cloud-object-storage:global:a/abac0df06b644a9cabc6e44f55b3880e:db530b95-6f50-4144-b0d0-6c341e23855f::"
```